ark/utils/deepcell_service_utils_test.py

```diff
@@ -0,0 +1,56 @@
+import pathlib
+import os
+import tempfile
+from zipfile import ZipFile
+import pytest
+
+from ark.utils.deepcell_service_utils import create_deepcell_output
+
+
+def mocked_run_deepcell(input_dir, output_dir, host, job_type):
+    pathlib.Path(os.path.join(output_dir, 'example_point_feature_0.tif')).touch()
```

Comment:
I think you'll either want to remove this file once the zip has been created, or create the zip without needing it in the first place. Right now we won't know if something is goign wrong in the unzip code, since we've already created the unzipped version of the file

Reply:
oh I missed that! so i'm now removing the file after it is added to the zip